tensorboard/webapp/widgets/histogram/histogram_v2_component.ts

```diff
@@ -139,8 +140,8 @@ export class HistogramV2Component
   }
 
   ngOnChanges() {
-    if (this.domInitialized) {
-      this.updateChart();
+    if (this.domVisible) {
```

Comment:
Let's drop the domVisible check here, now that `updateChartIfVisible` is clearly doing the check internally.